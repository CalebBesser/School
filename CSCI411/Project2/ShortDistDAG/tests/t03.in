13 20
0 3 2
0 12 3
1 0 4
1 6 5
2 11 1
3 5 1
3 7 5
4 8 5
5 7 3
5 8 2
6 3 9
7 8 4
8 2 3
8 11 7
9 1 10
9 10 3
9 12 7
10 6 8
12 4 6
12 5 7
printGraph
findShortPaths 1
