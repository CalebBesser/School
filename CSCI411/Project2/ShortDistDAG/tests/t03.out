Graph is:
0: 3/2	12/3	
1: 0/4	6/5	
2: 11/1	
3: 5/1	7/5	
4: 8/5	
5: 7/3	8/2	
6: 3/9	
7: 8/4	
8: 2/3	11/7	
9: 1/10	10/3	12/7	
10: 6/8	
11: 
12: 4/6	5/7	
Shortest distances from 1 to all reachable nodes are:
0 4
1 0
2 12
3 6
4 13
5 7
6 5
7 10
8 9
11 13
12 7
